diffulex_kernel/python/fused_moe_triton.py

```
@@ -0,0 +1,343 @@
+    w2: torch.Tensor,
+    topk_weights: torch.Tensor,
+    topk_ids: torch.Tensor,
+    top_k: int,
+    hidden_act: str = "silu",
+) -> torch.Tensor:
+    assert hidden_states.ndim == 2
+    assert hidden_act == "silu"
+
+    M, H = hidden_states.shape
+    E = w13.shape[0]
+    I = w13.shape[1] // 2
```

Ruff (0.15.7)

[error] Ambiguous variable name: `I`

(E741)